### ISSUE TITLE
How can I understand Query Rate and Indexing Rate (Cluster Metrics)

### ISSUE THREAD

**Issue by hintdesk** [acidvegas/elastop#9]
Hi,

can you please add more information in documentation (README) about these 2 metrics : Query Rate and Indexing Rate in Cluster Metrics? I don't understand the meaning of them.

For examples, the metrics of my cluster is

Search Queries: 28,305,380
Query Rate: 73,880/s
Total Query Time: 92433.6s
Avg Query Latency: 3.27ms

Index Operations: 978,978,179
Indexing Rate: 1,649,848/s
Total Index Time: 169822.3s
Avg Index Latency: 0.17ms

Thank you.

**Comment by acidvegas** [comment]
They are Elasticsearch cluster statistics derived from [GET /_nodes/stats](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-nodes-stats.html).

I will improve the documentation for all the items on elastop on my next commit.

**Comment by DavidGoodwin** [comment]
Yeah, I can't help but feel the query rate and indexing rate need dividing by the total query time / total index time numbers .... 

We're definitely not indexing 21 million documents a second .... 

![image](https://github.com/user-attachments/assets/f8d30dca-e574-48e5-83c9-d4e9c4fe87e7)

**Comment by acidvegas** [comment]
> Yeah, I can't help but feel the query rate and indexing rate need dividing by the total query time / total index time numbers ....
> 
> We're definitely not indexing 21 million documents a second ....
> 
> ![image](https://private-user-images.githubusercontent.com/203929/392779074-f8d30dca-e574-48e5-83c9-d4e9c4fe87e7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM1MDYzOTMsIm5iZiI6MTczMzUwNjA5MywicGF0aCI6Ii8yMDM5MjkvMzkyNzc5MDc0LWY4ZDMwZGNhLWU1NzQtNDhlNS04M2M5LWQ0ZTljNGZlODdlNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIwNlQxNzI4MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yYTJkNjA0NTA1MmU2MWIzMTViYjkzZTI5ZDk0NmQ2YjQ5YTRhYjk0YTNiOTBiMjMzYTlhNWY4ODZjYWM0YmIyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.LQLE_7FP9LluB3byv_AnSbI2t4zsrxAfGDc1oXWeTDc)

You are correct, my logic was wrong before. Let me know if the latest release looks better.

**Comment by neeej** [comment]
Still looks weird for me, this cluster is not really in use, it index like 20-40/s according to elastop (if you look at the bottom index list), but it doesn't actually index that much, and no ones really quering either, but the cluster metrics are high:
![image](https://github.com/user-attachments/assets/61a88174-c7dd-4c85-a223-40354f835645)

And this cluster index around 2.5-3k/s according to elastop, but it really index at around 400/s, so the cluster metrics here are also way off:
![image](https://github.com/user-attachments/assets/262becda-2a2d-4425-ae99-522cc639f2e9)
